act-rules-format.bs

```diff
@@ -51,7 +51,7 @@ In accessibility, there are often different technical solutions to make the same
   <p>If any one of these rules passes, the outcome of the composite rule for the table cell is "passed".</p>
 </div>
 
-The separation between atomic rules and composite rules creates a division of responsibility. Atomic rules test if web content is correctly implemented in a particular solution. Composite rules test if a combination of outcomes from atomic rules satisfy the conformance for an accessibility requirement.
+The separation between atomic rules and composite rules can be seen as a division of responsibility. Atomic rules typically test if web content is correctly implemented in a particular solution. Composite rules would test if a combination of outcomes from atomic rules satisfy the accessibility requirement, in part or as a whole.
```

Comment:
I'm not the grammar expert but "would" jumps out at me. I think it should be "can".

Reply:
I would simply delete the word 'would'.